sql-plugin/src/main/scala/com/nvidia/spark/rapids/FloatUtils.scala

```diff
@@ -63,21 +63,21 @@ object FloatUtils extends Arm {
     }
   }
 
-  def infinityToNulls(vec: ColumnVector): ColumnVector = {
-    def getInfinityVector: ColumnVector = {
-      if (vec.getType == DType.FLOAT64) {
-        ColumnVector.fromDoubles(Double.PositiveInfinity, Double.NegativeInfinity)
-      } else {
-        ColumnVector.fromFloats(Float.PositiveInfinity, Float.NegativeInfinity)
-      }
+  def getInfinityVector(vec: ColumnView): ColumnVector = {
```

Comment:
This should just take a `DType`.  Requiring a `ColumnView` is a bit excessive when all it needs is a type, and it makes for a confusing read of the caller's code, since it's not immediately clear why `getInfinityVector` would need the input column to do its job, whereas passing the type as a parameter is very clear.